go/consensus/tendermint/apps/roothash/state/round.go

```diff
@@ -13,13 +12,11 @@ import (
 type Round struct {
```

Comment:
Sorry to point this out again, but now that current block and finalized have been removed, why do we even have the round type? It looks like it just forwards everything to the `ExecutorPool`? Could we just move `ExecutorPool` and `CurrentBlockHeight` directly to `RuntimeState` and remove `Round`?

Reply:
fixed :-)